railgun/test/integration/helloworldgo.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: helloworld-go
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-samples/helloworld-go
+          env:
+            - name: TARGET
+              value: Go Sample v1
```

Comment:
We should deploy this service using the upstream corev1 api rather than raw yaml in the tests, e.g.:

```
svc := corev1.Service{...}
```

Reply:
I would say third party test sample varies per release, whats most convenient fits our needs. 

But if you really prefer clientgo style, we can do it as an improvement.

Reply:
Given that all the tests so far use "clientgo style", I would like to make this improvement so that we're doing these kinds of things one consistent way.